docs/example.md

```diff
@@ -35,6 +35,11 @@ Install dependencies with the following command:
 yarn install
```

Comment:
You need to run `yarn install` in the main folder too after this change, no?

Reply:
Can you update docs accordingly? I think it is more important to add that `install` needs to be run than that `yarn start` should be used (because the server is started automatically from xcode or gradle scripts, so that step isn't really required)

Reply:
done